src/main/resources/plugin.yml

```diff
@@ -2,7 +2,7 @@ name: ${project.name}
 main: com.github.maxopoly.MemeMana.MemeManaPlugin
 author: Maxopoly
 version: ${project.version}
-depend: [CivModCore, Donum]
+depend: [CivModCore, BanStick]
```

Comment:
I guess we wont need Donum if we go completly with virtual essence, so removing it should be fine.

Reply:
Teal still wants physical essence that you can take out and use for other stuff *excluding* pearl feeding, planning to re-add this whenever that happens. Want to get the core functionality working first.

Reply:
Yeah, sounds good.